### ISSUE TITLE
Document package:checks when/if it moves out of preview

### ISSUE THREAD

**Issue by parlough** [dart-lang/site-www#4009]
### What information is missing?

There is a new experimental testing package, [`package:checks`](https://github.com/dart-lang/test/tree/master/pkgs/checks) that presents an alternative to the traditional matchers. 

### How would you like us to fix this problem?

If/when this package moves beyond a preview stage, we should consider documenting it or at least noting its existence.

**Comment by parlough** [comment]
@natebosch I noticed `package:checks` had a first release this week. Is this something that's worth mentioning as an experimental option to try on dart.dev or has it not reached that point yet? We unfortunately don't have a lot of documentation about testing currently but we could at the very least link to it alongside `package:test` with a basic introduction.

\cc @MaryaBelanger

**Comment by natebosch** [comment]
I expect to do a wider announcement this week, so far it's just a quite release to gather some initial feedback. I'm not sure that it will be worth mentioning on dart.dev until it gets closer to a 1.0.0 release.

**Comment by parlough** [comment]
Sounds good, thanks for the extra context :)

**Comment by atsansone** [comment]
This appears to be far from 1.0.0. We can reopen / recreate when needed.

**Comment by parlough** [comment]
I'm going to close this until we're certain we want to document `package:checks`. Please reopen or open a new issue if/once it is ready to be called out and/or documented. Thanks!
